web/routes.go

```diff
@@ -82,6 +82,7 @@ func (sc *ServiceConfig) getRoutes() []rest.Route {
 		rest.Route{"GET", "/services/vhosts", gz(sc.authorizedClient(restGetVirtualHosts))},
 		rest.Route{"PUT", "/services/:serviceId/endpoint/:application/vhosts/*name", gz(sc.authorizedClient(restAddVirtualHost))},
 		rest.Route{"DELETE", "/services/:serviceId/endpoint/:application/vhosts/*name", gz(sc.authorizedClient(restRemoveVirtualHost))},
+		rest.Route{"POST", "/services/:serviceId/endpoint/:application/vhosts/:name/enable", gz(sc.authorizedClient(restVirtualHostEnable))},
```

Comment:
this is a nit-pick that we don't necessarily need to address, but strictly speaking we should not allow the user to create vhosts named "enable"